compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -348,15 +351,13 @@ private void defineType(BType type, BTypeSymbol tSymbol) {
     public void defineOperators() {
         // Binary arithmetic operators
         defineIntegerArithmeticOperations();
-        defineBinaryOperator(OperatorKind.ADD, xmlType, xmlType, xmlType);
-        defineBinaryOperator(OperatorKind.ADD, xmlType, stringType, xmlType);
-        defineBinaryOperator(OperatorKind.ADD, xmlType, charStringType, xmlType);
+
+        // XML arithmetic operators
+        defineXmlArithmaticOperations();
```

Comment:
Shall we rename to `defineXmlStringConcatanationOperations`

Reply:
Renamed.